setup.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+from distutils.core import setup
+
+setup(
+	name		= 'ffn',
+	version		= '0.1.0',
```

Comment:
Does this have to be specified? We are not planning on doing releases with this type of versioning scheme.